File: AkitaModel/Schema.cs
using System.Collections.Generic;

namespace Skyware.Lis.AkitaModel;

public class Schema
{

    public int Id { get; set; }

    public string Name { get; set; }

    public bool HasDoctor { get; set; }

    public bool HasFund1 { get; set; }

    public int? Fund1Id { get; set; }

    //public Fund fund1 { get; set; }

    public bool HasFund2 { get; set; }

    public int? Fund2Id { get; set; }

    //public Fund fund2 { get; set; }

    public string Field1 { get; set; }

    public string Field2 { get; set; }

    public string Field3 { get; set; }

    public bool IsActive { get; set; }

    public int? SaleTypeId { get; set; }

    public IEnumerable<Product> Items { get; set; }

    //public IEnumerable<Fund> enabledFunds;

    //public IEnumerable<PatientIdType> enabledPidTypes;

}

File: AkitaMotelTS/src/Schema.ts
/**
 * Skyware.Lis.AkitaModel
 */

import { Product } from './Product';

export class Schema {
  id!: number;
  name?: string;
  hasDoctor!: boolean;
  hasFund1!: boolean;
  fund1Id?: number;
  //fund1?: Fund;
  hasFund2!: boolean;
  fund2Id?: number;
  //fund2?: Fund;
  field1?: string;
  field2?: string;
  field3?: string;
  isActive!: boolean;
  saleTypeId?: number;
  items?: Product[];
  //enabledFunds?: Fund[];
  //enabledPidTypes?: PatientIdType[];
}
